pkg/redo/writer/file/file.go

```diff
@@ -229,6 +243,82 @@ func (w *Writer) IsRunning() bool {
 	return w.running.Load()
 }
 
+func (w *Writer) GetInputCh() chan writer.RedoEvent {
+	return w.inputCh
+}
+
+func (w *Writer) write(event writer.RedoEvent) error {
+	rl := event.ToRedoLog()
+	data, err := codec.MarshalRedoLog(rl, nil)
+	if err != nil {
+		return errors.WrapError(errors.ErrMarshalFailed, err)
+	}
+	w.AdvanceTs(rl.GetCommitTs())
+	_, err = w.Write(data)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func (w *Writer) SyncWrite(event writer.RedoEvent) error {
+	err := w.write(event)
+	if err != nil {
+		return err
+	}
+	err = w.Flush()
+	if err != nil {
+		return errors.Trace(err)
+	}
+	event.PostFlush()
+	return nil
+}
+
+func (w *Writer) encode(ctx context.Context) error {
+	ticker := time.NewTicker(time.Second * 1)
```

Comment:
it's better to use a config for the time of this ticker. And we need to choose a good default value for it later.